src/app/components/super-admin-login/super-admin-login.component.html

```diff
@@ -1,50 +1,57 @@
-<div class="login-page">
-  <mat-card>
-    <mat-card-header>SuperAdmin Login</mat-card-header>
-    <form [formGroup]="loginForm" (submit)="loginAdmin()">
-      <div class="login">
-        <div>
-          <mat-form-field appearance="outline">
-            <mat-label>Email Id</mat-label>
+<div>
+  <div class="admin-login">
+    <div class="image-section">
+      <img
+        src="https://img.lovepik.com/element/45009/2860.png_300.png"
+        alt="login image"
+      />
+    </div>
+    <div class="card">
```

Comment:
There is no `mat-card` that contains the card.
refer to https://material.angular.io/components/card/examples